apps/api/src/hyp3_api/validation.py

```diff
@@ -86,6 +86,28 @@ def check_dem_coverage(job, granule_metadata):
         raise GranuleValidationError(f'Some requested scenes do not have DEM coverage: {", ".join(bad_granules)}')
 
 
+def check_same_burst_ids(job, granule_metadata):
+    burst_ids = [g['name'].split('_')[1] for g in granule_metadata]
+    ref_burst_id = burst_ids[0]
+    sec_burst_id = burst_ids[1]
```

Comment:
More concise. Also better to use full variable names rather than single characters.
```suggestion
    ref_burst_id, sec_burst_id = [granule['name'].split('_')[1] for granule in granule_metadata]
```